app/controllers/home_controller.rb

```diff
@@ -14,7 +14,7 @@ def explore
   end
 
   def dashboard
-    @favorites = Workspace.accessible_by(current_ability, :update).favorites.order(updated_at: :desc).limit(3)
+    @favorites = current_user.favorited_workspaces.order(updated_at: :desc).limit(3)
```

Comment:
On a previous commit, I think you asked if we should keep checking accessible workspaces here? Maybe @ggeisler has thoughts about whether to:

- just hide non-accessible workspaces
- show... something.. to let the user know they no longer have access
- offer a broken link?

Reply:
Yeah, my [earlier comment](https://github.com/sul-dlss/mise/pull/165#discussion_r650314800) was ideally we should continue to track that a user favorited a given workspace, even if it has been made inaccessible to them (in case it later becomes accessible).

But definitely a second part of that is do we alert the user to the situation, and if so, how?

I don't know. Ideally, I guess some sort of notification might be helpful. But I'm also not sure it's a frequent or significant enough case to worry much about at this point. I'd suggest we go with "just hide non-accessible workspaces" and I can create a design-needed ticket to remember to think about this further and maybe do something more sophisticated in the future?

Reply:
OK! Will see to this later in the PM. Thanks, all.